### PR TITLE
Feature/rad 916

### PR DESCRIPTION
**Pull request by rogdevil**
added a Industry section with card view and responsiveness

### CODE REVIEW

File: .DS_Store
Comment (prakhar-ira) [inline .DS_Store:1]: delete this temporary file. Always make sure not to add this file when committing code stash this type of files

File: src/.DS_Store
Comment (prakhar-ira) [inline src/.DS_Store:1]: Delete

File: .DS_Store
Comment (rogdevil) [reply .DS_Store:1]: okey

File: src/.DS_Store
Comment (rogdevil) [reply src/.DS_Store:1]: okey

File: src/components/industry/industryPage1.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.industryPage1_parentContainer {
+    margin: 6.25rem 0 1.875rem 0;
+    color: var(--off-white-100);
+
+    h1{
+        font-size: 64px;
```
Comment (prakhar-ira) [inline src/components/industry/industryPage1.scss:6]: use rem convention. rem(64px);

File: src/components/industry/industryPage1.scss [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        font-style: normal;
+        font-weight: 700;
+        line-height: 74px;
+        span{
+            color: var(--primary-100);
+        }
+        margin-bottom: 80px;
```
Comment (prakhar-ira) [inline src/components/industry/industryPage1.scss:13]: use rem convention

File: src/components/industry/industryPage1.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
+import React from 'react'
```
Comment (prakhar-ira) [inline src/components/industry/industryPage1.tsx:1]: semicolons at end are missing everywhere

File: src/components/industry/industryPageCard.scss [hunk 4]
```diff
@@ -0,0 +1,47 @@
+.industryPageCard {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    max-width: 568px;
```
Comment (prakhar-ira) [inline src/components/industry/industryPageCard.scss:5]: use rem convention

File: src/components/industry/industryPageCard.tsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import './IndustryPageCard.scss'
+
+interface IndustryPageCardProps {
```
Comment (prakhar-ira) [inline src/components/industry/industryPageCard.tsx:4]: you declare this interface at global levels in types.ts and use interface here as well as above file

File: src/components/industry/industryPage1.scss [hunk 6]
```diff
@@ -0,0 +1,30 @@
+.industryPage1_parentContainer {
+    margin: 6.25rem 0 1.875rem 0;
+    color: var(--off-white-100);
+
+    h1{
```
[the remark this answers is not in the document]
Comment (rogdevil) [reply src/components/industry/industryPage1.scss:5]: done

File: src/components/industry/industryPage1.scss (hunk 1 above)
Comment (rogdevil) [reply src/components/industry/industryPage1.scss]: done

File: src/components/industry/industryPage1.scss [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+        font-size: 64px;
+        font-style: normal;
+        font-weight: 700;
+        line-height: 74px;
+        span{
```
[the remark this answers is not in the document]
Comment (rogdevil) [reply src/components/industry/industryPage1.scss:10]: done

File: src/components/industry/industryPage1.scss (hunk 2 above)
Comment (rogdevil) [reply src/components/industry/industryPage1.scss]: done

File: src/components/industry/industryPage1.tsx [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+      description: "Fintech companies leverage cutting-edge technologies such as artificial intelligence, big data analytics, blockchain, cloud computing, and mobile applications to offer a diverse array of financial products and services."
+    },
+  ]
+  return (
+    <Wrapper>
+      <div className='industryPage1_parentContainer'>
+        <h1>Software <span>solutions</span><br /> for every sector</h1>
+        <div className='cardsContainer'>
+          {
+            data.map((cardData, index) => <IndustryPageCard data={cardData} key={index} />)
```
[the remark this answers is not in the document]
Comment (rogdevil) [reply src/components/industry/industryPage1.tsx:51]: done

File: src/components/industry/industryPageCard.scss (hunk 4 above)
Comment (rogdevil) [reply src/components/industry/industryPageCard.scss:5]: done

File: src/components/industry/industryPageCard.scss [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+    min-width: 298px;
+    width: 100%;
+    flex-direction: column;
+    align-self: start;
+    &:nth-child(2n-1){
+        margin-top: 73px;
+    }
+
+    .cardImg{
+        padding: 20px 20px 0px 20px;
```
[the remark this answers is not in the document]
Comment (rogdevil) [reply src/components/industry/industryPageCard.scss:15]: done

File: src/components/industry/industryPageCard.scss [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+        background-color: var(--dark-gray-100);
+        max-height: 369px;
+        overflow: hidden;
+    }
+
+    .content{
+        .title {
+            font-size: 48px;
```
[the remark this answers is not in the document]
Comment (rogdevil) [reply src/components/industry/industryPageCard.scss:23]: done

File: src/components/industry/industryPageCard.scss [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+            font-style: normal;
+            font-weight: 700;
+            line-height: 58px;
+            margin-top: 48px;
+        }
+
+        .description {
```
[the remark this answers is not in the document]
Comment (rogdevil) [reply src/components/industry/industryPageCard.scss:30]: done

File: src/components/industry/industryPageCard.tsx (hunk 5 above)
Comment (rogdevil) [reply src/components/industry/industryPageCard.tsx:4]: done

**Pull request merged**